packages/select/src/Select.tsx

```diff
@@ -101,13 +102,13 @@ export default function Select({
 }: SelectProps) {
   const id = useIdAllocator({ prefix: 'select', id: idProp });
   const labelId = useMemo(
-    () => ariaLabelledby ?? `${id}-label`,
-    [ariaLabelledby, id],
+    () => (ariaLabel && !label ? undefined : ariaLabelledby ?? `${id}-label`),
+    [ariaLabelledby, ariaLabel, label, id],
   );
 
-  if (!label && !ariaLabelledby) {
+  if (!label && !ariaLabelledby && !ariaLabel) {
```

Comment:
just want to confirm that this isn't breaking anything a11y wise? (i don't expect that it would) cc @TheSonOfThomp

Reply:
I did this on purpose because an element shouldn't have both `aria-label` AND `aria-labelledby` - it should really just take in one of these props and one only